src/PolyhedralGeometry/visualization.jl

```diff
@@ -1,13 +1,79 @@
 
 Visualize a polyhedral object of dimension at most four (in 3-space).
 In dimensions up to 3 a usual embedding is shown.
 Four-dimensional polytopes are visualized as a Schlegel diagram, which is a projection onto one of the facets; e.g., see Chapter 5 of [Zie95](@cite).
 
 In higher dimensions there is no standard method; use projections to lower dimensions or try ideas from [GJRW10](@cite).
+
+# Extended help
+
+# Keyword Arguments
+
+## Colors
+
+Colors can be given as
+- a literal `String`, e.g. `"green"`.
+- a `String` of the format `"r g b"` where $r, g, b \in {0, \dots, 255}$ are integers corresponding to the R/G/B values of the color.
+- a `String` of the format `"r g b"` where $r, g, b \in [0, 1]$ are decimal values corresponding to the R/G/B values of the color.
+
+Possible arguments are:
+
+- `FacetColor`: Filling color of the polygons.
+- `EdgeColor`: Color of the boundary lines.
+- `PointColor`/`VertexColor`: Color of the spheres or rectangles representing the points.
+
+## Scaling and other gradient properties
+
+These arguments can be given as a floating point number:
+
+- `FacetTransparency`: Transparency factor of the polygons between 0 (opaque) and 1 (completely translucent).
+- `EdgeThickness`: Scaling factor for the thickness of the boundary lines.
+- `PointThickness`/VertexThickness`: Scaling factor for the size of the spheres or rectangles representing the points.
+
+## Camera
+
+These arguments can be given as a 3-element vector over floating point numbers:
+
+- `ViewPoint`: Position of the camera.
+- `ViewDirection`: Direction of the camera.
+
+## Appearance and Texts
+
+These arguments can be given as a string:
+
+- `FacetStyle`: If set to `"hidden"`, the inner area of the polygons are not rendered at all.
+- `FacetLabels`: If set to `"hidden"`, the facet labels are not displayed (in the most cases this is the default behavior). TODO
+- `EdgeStyle`: If set to `"hidden"`, the boundary lines are not rendered.
+- `Name`: The name of this visual object in the drawing.
+- `PointLabels`/`VertexLabels`: If set to `"hidden"`, no point labels are displayed.
+- `PointStyle`/`VertexStyle`: If set to `"hidden"`, neither point nor its label is rendered.
+- `LabelAlignment`: Defines the alignment of the vertex labels: `"left"`, `"right"` or `"center"`.
 """
-function visualize(P::Union{Polyhedron{T}, Cone{T}, PolyhedralFan{T}, PolyhedralComplex{T}}) where T<:Union{Float64, FieldElem}
+function visualize(P::Union{Polyhedron{T}, Cone{T}, PolyhedralFan{T}, PolyhedralComplex{T}, SubdivisionOfPoints{T}}; kwargs...) where T<:Union{Float64, FieldElem}
+  _prepare_visualization(P)
+  pmo = pm_object(P)
+  Polymake.visual(pmo; kwargs...)
+end
+
+function compose_visualization(P::Vararg{Union{Polyhedron{T}, Cone{T}, PolyhedralFan{T}, PolyhedralComplex{T}, SubdivisionOfPoints{T}}}; kwargs::Dict = Dict{Int, Nothing}()) where T<:Union{Float64, FieldElem}
```

Comment:
This vararg here is suboptimal as julia needs to recompile the function for each number of inputs and combination of types. I think a `Vector{<:Union{Polyhedron{T}, Cone{T}, PolyhedralFan{T}, PolyhedralComplex{T}, SubdivisionOfPoints{T}}` would perform better here.

Furthermore, the failing Aqua test is due to this signature. In the zero-argument case, the type parameter `T` has no occurrences in the signature.